scripts/importer.py

```
@@ -0,0 +1,359 @@
+    return 2  # Moderate
+
+
+def _truncate(value: str | None, max_len: int) -> str | None:
+    if value is None:
+        return None
+    return value[:max_len]
+
+
+def _load_staging() -> dict:
+    if not STAGING_FILE.exists():
+        print(f"No staging file found. Run 'fetch' first.")
```

Ruff (0.15.17)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)